autobahn/wamp_rawsocket_transport.hpp

```diff
@@ -0,0 +1,237 @@
+///////////////////////////////////////////////////////////////////////////////
+//
+// Copyright (c) Tavendo GmbH
```

Comment:
I am going to add a Topology copyright to every file as we have made significant modifications to all files. I will do it in a separate commit.

I think the preferred approach is to not include a copyright date as it generally just causes confusion to those not familiar with copyright law. The commit meta-data contains dates anyways. plus it will inevitably get out of sync as the code is updated.